sn_transfers/src/wallet/local_store.rs

```diff
@@ -821,8 +819,6 @@ mod tests {
 
         assert_eq!(0, recipient.wallet.balance().as_nano());
```

Comment:
We still create `received_cash_note_dir` above in this test.

Reply:
but its just `cash_notes` now